docs/content/快速上手/oracle-tutorial-zh.md

```diff
@@ -0,0 +1,141 @@
+# 演示: Oracle CDC 导入 Elasticsearch
+
+**1. 创建`docker-compose.yml`文件，内容如下所示:**
+
+```
+version: '2.1'
+services:
+  oracle:
+    image: yuxialuo/oracle-xe-11g-r2-cdc-demo:v1.0
+    ports:
+      - "1521:1521"
+  elasticsearch:
+    image: elastic/elasticsearch:7.6.0
+    environment:
+      - cluster.name=docker-cluster
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - discovery.type=single-node
+    ports:
+      - "9200:9200"
+      - "9300:9300"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+      nofile:
+        soft: 65536
+        hard: 65536
+  kibana:
+    image: elastic/kibana:7.6.0
+    ports:
+      - "5601:5601"
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+``` 
+该 Docker Compose 中包含的容器有:
+- Oracle: Oracle 11g, 已经预先创建了 `products` 和 `orders`表，并插入了一些数据. 
+- Elasticsearch: `orders` 表将和 `products` 表进行join，join的结果写入Elasticsearch中
+- Kibana: 可视化 Elasticsearch 中的数据
+
+在 docker-compose.yml 所在目录下运行如下命令以启动所有容器：
+```shell
+docker-compose up -d
+```
+该命令会以 detached 模式自动启动 Docker Compose 配置中定义的所有容器。
+你可以通过 docker ps 来观察上述的容器是否正常启动了。 也可以访问 http://localhost:5601/ 来查看 Kibana 是否运行正常。
+另外可以通过如下命令停止所有的容器：
+
+```shell
+docker-compose down
+````
+
+**2. 下载以下 jar 包到 `<FLINK_HOME>/lib/`:**
+
+*下载链接只在已发布的版本上可用*
+
+- [flink-sql-connector-elasticsearch7_2.11-1.13.2.jar](https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-elasticsearch7_2.11/1.13.2/flink-sql-connector-elasticsearch7_2.11-1.13.2.jar)
+- [flink-sql-connector-oracle-cdc-2.2-SNAPSHOT.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-oracle-cdc/2.2-SNAPSHOT/flink-sql-connector-oracle-cdc-2.2-SNAPSHOT.jar)
+
+**3. 然后启动 Flink 集群，再启动 SQL CLI:**
+
+```sql
+-- Flink SQL
+-- checkpoint every 3000 milliseconds                       
+Flink SQL> SET execution.checkpointing.interval = 3s;
+
+Flink SQL> CREATE TABLE products (
+    ID INT,
+    NAME STRING,
+    DESCRIPTION STRING,
+    PRIMARY KEY (ID) NOT ENFORCED
+  ) WITH (
+    'connector' = 'oracle-cdc',
+    'hostname' = 'localhost',
+    'port' = '1521',
+    'username' = 'flinkuser',
+    'password' = 'flinkpw',
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)